engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetBlend.java

```diff
@@ -15,48 +15,55 @@
  */
 package org.terasology.rendering.dag.stateChanges;
 
+import static org.lwjgl.opengl.GL11.GL_BLEND;
 import org.terasology.rendering.dag.RenderPipelineTask;
 import org.terasology.rendering.dag.StateChange;
+import org.terasology.rendering.dag.tasks.DisableCapabilityTask;
 import org.terasology.rendering.dag.tasks.EnableCapabilityTask;
 
 /**
  * TODO: Add javadocs
  */
-public class EnableCapability implements StateChange {
-    private int capability;
-    private EnableCapabilityTask task;
+public final class SetBlend extends SetCapability {
+    private static final int CAPABILITY = GL_BLEND;
+    private static StateChange defaultInstance = new SetCullFace(false);
```

Comment:
Perhaps I'm not understanding or perhaps I wasn't clear enough:

why is the default instance of SetBlend an instance of SetCullFace?